NuGet.config

```diff
@@ -5,5 +5,6 @@
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="powershell-core" value="https://powershell.myget.org/F/powershell-core/api/v3/index.json" />
+    <add key="myget-msal" value="https://www.myget.org/F/aad-clients-nightly/api/v3/index.json" />
```

Comment:
Is this stuff not published to NuGet yet?

Reply:
@markcowl some of the builds I was working with weren't, but now everything is pushed up to NuGet. I'll removed this reference.